viewer/index.html

```diff
@@ -0,0 +1,224 @@
+              <ul class="nav list-group list-group-flush wl__sidebar-feature-name-list"></ul>
+            </div>
+          </div>
+        </div>
+
+        <div class="main">
+          <div class="wl-table-wrap d-none" id="table-content">
+            <div class="wl-table">
+              <div class="wl-table-heading">
+                <div class="wl-table-row wl-table-row--bottom-shadow">
+                  <div class="wl-table-head">Feature</div>
+                  <div class="wl-table-head">Frequent items</div>
+                  <div class="wl-table-head">Inf. feature type</div>
+                  <div class="wl-table-head text-end">Total count</div>
+                  <div class="wl-table-head text-end">Null fraction</div>
+                  <div class="wl-table-head text-end">Est. unique values</div>
+                  <div class="wl-table-head">Data type</div>
+                  <div class="wl-table-head text-end">Data type count</div>
+                  <div class="wl-table-head text-end">Mean</div>
+                  <div class="wl-table-head text-end">Std. dev</div>
+                  <div class="wl-table-head text-end">Min</div>
+                  <div class="wl-table-head text-end">First quantile</div>
+                  <div class="wl-table-head text-end">Median</div>
+                  <div class="wl-table-head text-end">Third quantile</div>
+                  <div class="wl-table-head text-end">Max</div>
+                </div>
+              </div>
+              <ul class="wl-table-body wl__table-body"></ul>
+            </div>
+          </div>
+
+          <div class="wl-table__text-message" id="table-message">
+            <p>Please select and load a JSON profile from your local drive.</p>
+          </div>
+
+          <div class="wl-property-panel">
+            <button type="button" class="wl-property-panel__button btn-close" aria-label="Close"></button>
+            <h5 class="wl-property-panel__title"></h5>
+            <div>
+              <table class="wl-property-panel__table">
+                <thead class="wl-property-panel__table-head">
+                  <tr class="wl-property-panel__table-tr">
+                    <th class="wl-property-panel__table-th">Item</th>
+                    <th class="wl-property-panel__table-th wl-property-panel__table-th-profile"></th>
+                  </tr>
+                </thead>
+                <tbody class="wl-property-panel__frequent-items">
+                  <!-- CONTENT IS GENERATED -->
+                </tbody>
+              </table>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <!-- Bootstrap core JavaScript
+    ================================================== -->
+    <!-- Placed at the end of the document so the pages load faster -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="https://d3js.org/d3.v6.min.js"></script>
+    <script
```

Comment:
@valer-whylabs  -- Just realized; we can't call out to external libraries like this, so we should download the minified versions of them from
https://jquery.com/download/
https://d3js.org/
and
https://getbootstrap.com/docs/4.3/getting-started/download/

Yet another reason why it's good that we kept the dependencies to a minimum

Reply:
Should we include the full libraries or extract the specific parts we need for the html viewer?

Reply:
It's all local; I'm not sure what the size is nor how difficult it is to extract